sample/src/androidTest/java/com/schibsted/spain/barista/sample/NavigationDrawerActivityTest.java

```diff
@@ -33,4 +38,26 @@ public void openAndCloseDrawer() {
     assertNotDisplayed("menu item");
     assertDrawerIsClosed(R.id.drawer);
   }
+
+  @Test
+  public void openAndCloseTheOnlyDrawer() {
+    openDrawer();
+    assertDisplayed("menu item");
+    assertDrawerIsOpen();
+
+    closeDrawer();
+    assertNotDisplayed("menu item");
+    assertDrawerIsClosed();
+  }
+
+  @Test
+  public void openAndCloseDrawerAtRight() {
```

Comment:
Suggestion: `openAndCloseDrawerAtEnd`. In addition, what happens if you use `GravityCompat.RIGHT`? Do you think we need to test it? If no, go on :·D

Reply:
Yeh... I had doubts about it, to me "AtEnd" sounds like it's referring to a time dimension, not gravity. How about `openAndCloseDrawerAtEndGravity`?

About the second topic, you should expect it to work on the same gravity that your layout has. If you use `right` in your layouts, use `Gravity.RIGHT`. If you use `end`, use `GravityCompat.END`. It's not important which one you use, as long as you use the same in your app. Don't see a need for a special test. But thanks for raising the question! We could have missed a special case.

Reply:
Noice!